### ISSUE TITLE
IDE0079 should not warn when #Disable Warning IDE0049 is suppressing issue

### ISSUE THREAD

**Issue by paul1956** [dotnet/roslyn#50892]
**Version Used**: 
16.9 Preview 3
**Steps to Reproduce**:
```VB
<ComponentModel.EditorBrowsable(ComponentModel.EditorBrowsableState.Never)>
Public NotInheritable Class NativeTypes

    Public Const OBJID_VSCROLL As Long = &HFFFFFFFB

#Disable Warning IDE0049 ' Simplify Names
    Public Const WM_MOUSELEAVE As Int32 = &H2A3
    Public Const WM_MOUSEMOVE As Int32 = &H200
#Enable Warning IDE0049 ' Simplify Names

End Class
```
![image](https://user-images.githubusercontent.com/12789369/106232816-cbb5f900-61a9-11eb-85fa-caf573e6c0af.png)

**Expected Behavior**:
IDE0079 should not be shown, if you remove them you get
![image](https://user-images.githubusercontent.com/12789369/106232933-22233780-61aa-11eb-9f38-c92f05969633.png)

**Actual Behavior**:
Messages above

With file open in Visual Studio, when you build the message disappears until you close the file in IDE then it comes back. If you don't close the message does not reappear until you restart.

If you suppress with Attribute as below the issue does not happen
```VB
<ComponentModel.EditorBrowsable(ComponentModel.EditorBrowsableState.Never)>
Public NotInheritable Class NativeTypes

    Public Const OBJID_VSCROLL As Long = &HFFFFFFFB

    <CodeAnalysis.SuppressMessage("Style", "IDE0049:Simplify Names", Justification:="Compatibility with API")>
    Public Const WM_MOUSELEAVE As Int32 = &H2A3
    <CodeAnalysis.SuppressMessage("Style", "IDE0049:Simplify Names", Justification:="Compatibility with API")>
    Public Const WM_MOUSEMOVE As Int32 = &H200

    Public Const WM_NCRBUTTONDOWN As Integer = &HA4

End Class
```

**Comment by huoyaoyuan** [comment]
Related to https://github.com/dotnet/roslyn/issues/49925

**Comment by paul1956** [comment]
@huoyaoyuan I looked at that one, it is one for projects targeting multiple Frameworks, this one only targets net5.0 and only just started happening with Preview 3.0.

**Comment by mavasani** [comment]
I am unable to repro this on the latest internal 16.10 dogdood builds.

**Comment by mavasani** [comment]
Closing based on the last comment. Please reopen if you still see this.
